community/cypher/docs/cypher-docs/src/docs/dev/statistics.asciidoc

```diff
@@ -30,6 +30,11 @@ If background sampling is turned off, make sure to trigger manual sampling when
 `index_sampling_update_percentage`::
 Controls how large portion of the index has to have been updated before a new sampling run is triggered.
 
+`dbms.cypher.statistics_divergence_threshold`::
+Controls how much the above statistical information is allowed to change before an execution plan is considered stale.
+If the relative change in any of statistics is larger than this threshold, the plan will thrown away and a new one will
```

Comment:
In docs, the code style says that you should use a single line per sentence, no more, no less. The line can be arbitrarily long